x-pack/plugins/uptime/public/components/fleet_package/browser/advanced_fields.tsx

```diff
@@ -39,6 +41,73 @@ export const BrowserAdvancedFields = () => {
       data-test-subj="syntheticsBrowserAdvancedFieldsAccordion"
     >
       <EuiSpacer size="m" />
+      <EuiDescribedFormGroup
+        title={
+          <h4>
+            <FormattedMessage
+              id="xpack.uptime.createPackagePolicy.stepConfigure.browserAdvancedSettings.filtering.title"
+              defaultMessage="Synthetics agent filtering"
+            />
+          </h4>
+        }
+        description={
+          <FormattedMessage
+            id="xpack.uptime.createPackagePolicy.stepConfigure.browserAdvancedSettings.filtering.description"
+            defaultMessage="Filter tests using match or tags options."
+          />
+        }
+      >
+        <EuiSpacer size="s" />
+        <EuiFormRow
+          label={
+            <FormattedMessage
+              id="xpack.uptime.createPackagePolicy.stepConfigure.browserAdvancedSettings.journeyFiltersMatch.label"
+              defaultMessage="Filter match"
+            />
+          }
+          labelAppend={<OptionalLabel />}
+          helpText={
+            <FormattedMessage
+              id="xpack.uptime.createPackagePolicy.stepConfigure.browserAdvancedSettings.journeyFiltersMatch.helpText"
+              defaultMessage="Run only journeys with a name or tags that matches the provided glob."
+            />
+          }
+        >
+          <EuiFieldText
+            value={fields[ConfigKeys.JOURNEY_FILTERS_MATCH]}
+            onChange={(event) =>
+              handleInputChange({
+                value: event.target.value,
+                configKey: ConfigKeys.JOURNEY_FILTERS_MATCH,
+              })
+            }
+            data-test-subj="syntheticsBrowserJourneyFiltersMatch"
+          />
+        </EuiFormRow>
+        <EuiFormRow
+          label={
+            <FormattedMessage
+              id="xpack.uptime.createPackagePolicy.stepConfigure.browserAdvancedSettings.journeyFiltersTags.label"
+              defaultMessage="Filter tags"
+            />
+          }
+          labelAppend={<OptionalLabel />}
+          helpText={
+            <FormattedMessage
+              id="xpack.uptime.createPackagePolicy.stepConfigure.browserAdvancedSettings.journeyFiltersTags.helpText"
+              defaultMessage="Run only journeys with the given tag(s), or globs."
```

Comment:
```suggestion
              defaultMessage="Run only journeys with the given tags, or globs."
```

Think we can skip this since globs is plural.